src/js/entities-service/entities/program-flows.js

```diff
@@ -16,10 +24,10 @@ const _Model = BaseModel.extend({
   validate({ name }) {
     if (!_.trim(name)) return 'Flow name required';
   },
-  getRole() {
-    const roleId = this.get('_role');
-    if (!roleId) return;
-    return Radio.request('entities', 'roles:model', roleId);
+  getOwner() {
+    const owner = this.get('_owner');
+    if (!owner || owner.type !== 'roles') return;
```

Comment:
is the type check necessary?

Reply:
No, it'll never be anything other than 'roles'